crates/client/rpc-core/Cargo.toml

```diff
@@ -33,6 +33,7 @@ jsonrpsee = { workspace = true, features = [
 mp-block = { workspace = true }
 mp-digest-log = { workspace = true }
 num-bigint = { workspace = true }
+pallet-starknet = { workspace = true }
```

Comment:
We don't want the rpc to depends on the pallet. Move `PredeployedAccount` type under `primitive/`

Reply:
should I create a new crate under `primitive` ? Otherwise, which crate would `predeployedAccount` fit in ?

Reply:
maybe a `genesis-config` crate. And you can maybe moves other types in it. Types that are in pallet/genesis-loader. 

But the From impl should stay into the pallet create